src/server/modules/bulk/services/BulkService.ts

```diff
@@ -67,6 +68,28 @@ export class BulkService implements interfaces.BulkService {
             noParsingError
 
           if (!validRow) {
+            switch (validRow) {
+              case acceptableLinkCount:
+                schema.errorMessage = `File exceeded ${BULK_UPLOAD_MAX_NUM} original URLs to shorten`
+                break
+              case onlyOneColumn:
+                schema.errorMessage = `${rowData} contains more than one column of data`
```

Comment:
nit, can we add the row number?

Reply:
yeap that would be good. added in [6af608b](https://github.com/opengovsg/GoGovSG/pull/1993#:~:text=to%20error%20message-,6af608b,-Merge%20state)